sonic-psud/tests/mock_platform.py

```diff
@@ -282,7 +282,9 @@ def __init__(self,
                  temp_high_th=50.0,
                  voltage_low_th=11.0,
                  voltage_high_th=13.0,
-                 replaceable=True):
+                 replaceable=True,
+                 in_current=0.72,
```

Comment:
Just for my knowledge, from where did you get these values for V and I?

Reply:
Do you mean the values 0.72 and 220.25?

Reply:
Yeah.

Reply:
I took it from a real switch - 
for in_voltage - /sys/devices/platform/mlxplat/i2c_mlxcpld.2/i2c-2/i2c-5/5-0059/hwmon/hwmon6/in1_input
and for in_current - /sys/devices/platform/mlxplat/i2c_mlxcpld.2/i2c-2/i2c-5/5-0059/hwmon/hwmon6/curr1_input